frontend/components/InfoBanner/_styles.scss

```diff
@@ -1,7 +1,45 @@
 .info-banner {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
```

Comment:
there is a UI bug I see on host details page info banner where the banner extends past the main content container. 

![image](https://user-images.githubusercontent.com/1153709/212942611-50a57032-86ba-452c-be23-ee88ff9c714f.png)

tried playing with box-sizing property and that seemed to fix it but worth checking if this an issue other places or leads to more issues when we change to box-sizing

Reply:
Looking into it thank you!